fbw-a32nx/src/wasm/extra-backend-a32nx/src/Gauge_Extra_Backend.cpp

```diff
@@ -49,7 +49,7 @@ AircraftPresets aircraftPresets(msfsHandler, AircraftPresetProcedures_A32NX::air
  * https://docs.flightsimulator.com/html/Content_Configuration/SimObjects/Aircraft_SimO/Instruments/C_C++_Gauges.htm?rhhlterm=_gauge_callback&rhsearch=_gauge_callback
  */
 extern "C" {
-[[maybe_unused]] MSFS_CALLBACK bool Gauge_Extra_Backend_gauge_callback([[maybe_unused]] FsContext ctx, int svcId, void* pData) {
+MSFS_CALLBACK bool Gauge_Extra_Backend_gauge_callback([[maybe_unused]] FsContext ctx, int svcId, void* pData) {
```

Comment:
[[maybe_unused]] 

Can this be cleaned up at this moment? Or for now are we still uncertain.

Reply:
I typically let my IDE guide me on this. 

Previously it showed a warning for the whole function which it does not any longer - probably was a change in the compiler. There were never any warnings when compiled via the dev-env for the first `[[maybe_unused]]` - the one I removed.

The second `[[maybe_unused]]`is necessary as indeed the ctx is not used:
![image](https://github.com/flybywiresim/aircraft/assets/16833201/6fd75999-4c7e-46ad-adb9-9a961d85db86)

I could alternatively just remove the var name which basically means the same thing as `[[maybe_unused]]`:
```cpp
MSFS_CALLBACK bool Gauge_Extra_Backend_gauge_callback(FsContext, int svcId, void* pData) {
```

If you prefer the second let me know - doesn't matter either way for me. 
It is just a bit clearer with the `[[maybe_unused]]`that the var is intentionally not used.

Reply:
Ok, this is fine, as long as it is intentional.